artisan.plugin.zsh

```diff
@@ -19,13 +19,14 @@ artisan() {
 
     if [[ $1 = "make:"* && $ARTISAN_OPEN_ON_MAKE_EDITOR != "" ]]; then
         # Find and open files created by artisan
+        _artisan_start_time=`date +%s`
```

Comment:
This will need to go before the `php $_artisan $*` call

Reply:
> This will need to go before the `php $_artisan $*` call

@jessarcher yeah that makes sense 🤦 fixed now!